src/status_im2/contexts/quo_preview/notifications/toast.cljs

```diff
@@ -86,22 +86,19 @@
                  :duration   3000}])}
             "update above toast"]])))))
 
-(defn preview-toasts
+(defn view
   []
-  (fn []
-    [preview/preview-container
-     [rn/view
-      {:background-color "#508485"
-       :flex-direction   :column
-       :justify-content  :flex-start
-       :height           300}]
-     [into
-      [rn/view
-       {:flex    1
-        :padding 16}]
-      [^{:key :basic} [toast-button-basic]
-       ^{:key :with-undo-action} [toast-button-with-undo-action]
-       ^{:key :with-multiline} [toast-button-multiline]
-       ^{:key :30s-duration} [toast-button-30s-duration]
-       ^{:key :upsert}
-       [update-toast-button]]]]))
+  [preview/preview-container
+   {:component-container-style
+    {:flex-direction  :column
+     :justify-content :flex-start}}
+   [into
+    [rn/view
+     {:flex    1
+      :padding 16}]
+    [^{:key :basic} [toast-button-basic]
```

Comment:
Since you're updating the preview, consider removing the manual React keys because it's unnecessary if the children are hardcoded.

```clojure
[rn/view {:style {:flex 1 :padding 16}}
 [toast-button-basic]
 [toast-button-with-undo-action]
 [toast-button-multiline]
 [toast-button-30s-duration]
 [update-toast-button]]
```

Reply:
thank you a lot